code/_onclick/item_attack.dm

```diff
@@ -146,7 +146,7 @@
 		return TRUE //successful attack
 
 /mob/living/simple_animal/attacked_by(obj/item/I, mob/living/user)
-	if(attack_threshold_check(I.force, I.damtype, MELEE, FALSE))
+	if(!attack_threshold_check(I.force, I.damtype, MELEE, FALSE))
```

Comment:
Having this proc return FALSE means it sends a little message about failing to do damage, is that something we want? would it be better to make a separate proc to handle the behavior/check you want?

Reply:
Yes, it is something we want- this is the same message that was sent before my PR if you failed to meet a simplemob's damage threshold or if you tried to use a stamina melee weapon on it.